modules/ROOT/pages/email/email-list.adoc

```diff
@@ -39,7 +39,7 @@ the xref:email/email-documentation.adoc[Email Connector Technical Reference].
 
 === Secured Connections
 
-The IMAP and POP3 configurations provide a secured connection type to work over the secured versions of the protocols (POP3S and IMAPS). This type allows you to configure a TLS context that enables SSL/TLS encryption.
+The IMAP and POP3 configurations provide a secured connection type to work over the secured versions of the protocols (POP3S and IMAPS). These connection types allow you to configure a TLS context that enables SSL/TLS encryption.
```

Comment:
The first sentence sounds redundant to me, but I am not quite sure I have an alternative that would have the exact same meaning. How's this?
```suggestion
The IMAP and POP3 configurations enable connections that are secured by the secured versions of the POP3S and IMAPS protocols. These connections enable you to configure a TLS context that enables both SSL and TLS encryption.
```